src/core/SablierNFTDescriptor.sol

```diff
@@ -347,14 +347,14 @@ contract SablierV2NFTDescriptor is ISablierV2NFTDescriptor {
     /// @dev Reverts if the symbol is unknown.
     function mapSymbol(IERC721Metadata sablier) internal view returns (string memory) {
         string memory symbol = sablier.symbol();
-        if (symbol.equal("SAB-V2-LOCKUP-LIN")) {
+        if (symbol.equal("SAB-LOCKUP-LIN") || symbol.equal("SAB-V2-LOCKUP-LIN")) {
```

Comment:
Review required @sablier-labs/solidity.

Reply:
Good question,  and I also think that we should have compatible with previous versions

Reply:
Sounds good. Thanks for confirming.